src/concrete/ml/onnx/ops_impl.py

```diff
@@ -946,11 +949,23 @@ def numpy_greater(
     Args:
         x (numpy.ndarray): Input tensor
         y (numpy.ndarray): Input tensor
+        **kwargs (Dict[str, Optional[Callable]]): Contains the `AutoRounder` which determines
```

Comment:
I think, though not 100% sure, you can avoid using `kwargs` and instead use the python kwargs delimiter like this:

```
def numpy_greater(
    x: numpy.ndarray,
    y: numpy.ndarray, 
    *, 
    auto_rounder: Optional[Callable],
)
```

this allows you to call this function with a `kwargs` dictionary

Reply:
Indeed, it's clearer, but it makes the argument `auto_rounder` mandatory and these functions are used elsewhere.